libs/features/mas/web-components/src/global.css.js

```diff
@@ -29,6 +29,8 @@ styles.innerHTML = `
     --consonant-merch-card-cta-font-size: 15px;
 
     /* headings */
+    --consonant-merch-card-heading-xxs-font-size: 16px;
```

Comment:
Also, let's not use `--consonant` prefix for CCD cards are they are not meant to be one.

Reply:
Removed consonant prefix